### PR TITLE
Update from upstream

### PR DESCRIPTION
**Pull request by amdbrujin**
This PR updates the project from the upstream repository

### CODE REVIEW

**Comment by amdbrujin** [comment]
@yenneferofvengerberg would you please check this? Nearly everything is safe for contribution to the original repository

File: src/DotNetOutdated.Core/DotNetOutdated.Core.csproj
Comment (yenneferofvengerberg) [inline src/DotNetOutdated.Core/DotNetOutdated.Core.csproj:1]: We should target corefx 8.0 and 9.0 only. Can you drop the others

File: test/DotNetOutdated.Tests/DotNetOutdated.Tests.csproj
Comment (yenneferofvengerberg) [inline test/DotNetOutdated.Tests/DotNetOutdated.Tests.csproj:1]: Here the same as above

**Review by yenneferofvengerberg — CHANGES_REQUESTED** [review]

File: src/DotNetOutdated.Core/DotNetOutdated.Core.csproj
Comment (amdbrujin) [reply src/DotNetOutdated.Core/DotNetOutdated.Core.csproj:1]: Ok, I had another PR for this. I squash it.

**Comment by amdbrujin** [comment]
@yenneferofvengerberg done

**Review by yenneferofvengerberg — APPROVED** [review]
LGTM

**Pull request merged**